deployment/helm/templates/deployment.yaml

```diff
@@ -57,13 +57,17 @@ spec:
           - "--metric-host=0.0.0.0"
           - "--github-client-id-file=/secrets/app/client_id"
           - "--github-client-secret-file=/secrets/app/client_secret"
+          - "--github-app-client-id-file=/secrets/github-app/github_app_client_id"
+          - "--github-app-client-secret-file=/secrets/github-app/github_app_client_secret"
```

Comment:
can't we get these from an environment variable?

Reply:
We can, I decided to follow the same pattern that we use for the GitHub OAuth client ID and secret